.changeset/sour-singers-glow.md

```diff
@@ -0,0 +1,43 @@
+---
+'@keystone-6/example-document-field-customisation-keystone-server': minor
+'@keystone-6/example-nextjs-keystone-two-servers-backend': minor
+'@keystone-6/example-extend-graphql-schema-graphql-tools': minor
+'@keystone-6/example-extend-graphql-schema-graphql-ts': minor
+'@keystone-6/test-projects-crud-notifications': minor
+'@keystone-6/example-extend-graphql-subscriptions': minor
+'@keystone-6/example-extend-graphql-schema-nexus': minor
+'@keystone-6/example-custom-admin-ui-navigation': minor
+'@keystone-6/example-with-custom-session-validation': minor
+'@keystone-6/test-projects-live-reloading': minor
+'@keystone-6/example-custom-admin-ui-pages': minor
+'@keystone-6/example-custom-admin-ui-logo': minor
+'@keystone-6/example-extend-prisma-schema': minor
+'@keystone-6/example-usecase-task-manager': minor
+'@keystone-6/redis-session-store-example': minor
+'@keystone-6/example-extend-express-app': minor
+'@keystone-6/example-usecase-versioning': minor
+'@keystone-6/example-custom-field-view': minor
+'@keystone-6/example-usecase-ecommerce': minor
+'@keystone-6/test-projects-basic': minor
+'@keystone-6/example-nextjs-keystone': minor
+'@keystone-6/example-default-values': minor
+'@keystone-6/example-document-field': minor
+'@keystone-6/example-graphql-ts-gql': minor
+'@keystone-6/example-usecase-basic': minor
+'@keystone-6/example-roles': minor
+'@keystone-6/example-virtual-field': minor
+'@keystone-6/example-assets-local': minor
+'@keystone-6/example-custom-field': minor
+'@keystone-6/example-usecase-blog': minor
+'@keystone-6/example-assets-s3': minor
+'@keystone-6/singleton': minor
+'@keystone-6/example-testing': minor
+'@keystone-6/example-limits': minor
+'@keystone-6/example-script': minor
```

Comment:
Can you please remove all the examples from this

Reply:
Done